osc-server/src/main/java/org/osc/core/broker/util/VirtualizationConnectorUtil.java

```diff
@@ -194,7 +197,11 @@ private void delayedInit() {
                 this.keystoneApi.listProjects();
 
             } catch (Exception exception) {
-                errorTypeException = new ErrorTypeException(exception, ErrorType.PROVIDER_EXCEPTION);
+				if (exception instanceof ConnectionException || exception instanceof AuthenticationException) {
```

Comment:
why is authentication exception a connect exception? we should provide the correct exceptions and messages for connection vs auth exceptions.